ddl/constant.go

```diff
@@ -80,5 +77,6 @@ const (
 		start_time bigint,
 		state_update_time bigint,
 		meta longblob,
+		summary json,
```

Comment:
Why replace ExecutionInfo with summary?

Reply:
It is the short term of "subtask execution summary". I think both are fine.